sage-ios/SAGE/SAGE/SignUpRequestTableViewCell.swift

```diff
@@ -0,0 +1,116 @@
+//
+//  CheckinRequestTableViewCell.swift
+//  SAGE
+//
+//  Created by Sameera Vemulapalli on 11/14/15.
+//  Copyright © 2015 Cal Blueprint. All rights reserved.
+//
+
+import UIKit
+import FontAwesomeKit
+
+class SignUpRequestTableViewCell: UITableViewCell {
+    
+    var userPicture = UIImageView()
+    var userName = UILabel()
+    var schoolAndHours = UILabel()
+    var checkButton = UIButton()
+    var xButton = UIButton()
+    
+    var userID: Int?
+    
+    override init(style: UITableViewCellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        self.contentView.addSubview(self.userPicture)
+        self.contentView.addSubview(self.userName)
+        self.contentView.addSubview(self.schoolAndHours)
+        self.contentView.addSubview(self.checkButton)
+        self.contentView.addSubview(self.xButton)
+        self.selectionStyle = .None
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    func configureWithUser(user: User) {
+        self.userID = user.id
+        self.userPicture.setImageWithUser(user)
+        self.userPicture.layer.cornerRadius = UIConstants.userImageSize/2
```

Comment:
this setup method that will stay constant should be in some setupSubviews method, its better to separate config and layout code